### ISSUE TITLE
iPAS AI 應用規劃師科目3練習 - 2026-09-03

### ISSUE THREAD

**Issue by JohnnyLai92** [JohnnyLai92/ipas#18]
# iPAS AI 應用規劃師科目3練習 - 2026-09-03（第 8 輪）

> 科目3：機器學習技術與應用｜本輪側重 **L234 機器學習治理**，並鞏固 **L231 基礎數學**（貝氏基率、矩陣維度、學習率）等歷次易失分觀念。

## 練習說明

- 本次共 **10 題**，涵蓋單選 4 題、多選 3 題、情境分析 2 題、指標計算 1 題。
- 作答方式：**直接點選每個選項前的核取方塊**即可作答，不需留言、不需進入編輯畫面、不需另外填寫「答案欄位」。
- 多選題請依標題指示的勾選數量作答（例如「可勾選 3 項」即需勾滿 3 項）。
- 每題僅標示題型與考點 L 代碼，**未提供正確答案**，請自行判斷。
- 全部勾選完成後，請將本 issue 的 label 改為 `ai-analysis`，系統即會進行批改與錯題分析。

## 第 1 題〔單選〕L23401 資料與模型治理

某研究單位欲釋出一份資料集供外部單位合作分析，目標是即使面對重新識別（re-identification）攻擊，也無法將任一筆資料還原至特定個人。下列敘述何者最正確？

- [ ] (A) 只要用代碼取代姓名、身分證等直接識別欄位（假名化），即已達成不可逆的匿名化
- [ ] (B) 匿名化的目標是使資料無法（或極難）重新連結到特定個人，且處理過程原則上不可逆
- [ ] (C) 只要刪除「姓名」一個欄位，就完成匿名化，其餘欄位組合不需考量
- [ ] (D) 資料一經去識別化，即永久不受任何隱私或合規規範約束

---

## 第 2 題〔單選〕L23402 AI 風險與倫理

某團隊為降低偏見，將「性別」欄位自訓練資料中移除，但模型對不同性別族群仍產生明顯差異的結果。最可能的原因為何？

- [ ] (A) 移除敏感欄位後模型必然公平，觀察到的差異只是隨機誤差
- [ ] (B) 其他特徵（如居住地區、消費樣態、就讀科系）可能與性別高度相關，形成代理變數（proxy variable）而間接帶入偏見
- [ ] (C) 只要模型整體 Accuracy 夠高，就代表沒有偏見
- [ ] (D) 偏見只可能來自演算法本身，與訓練資料無關

---

## 第 3 題〔單選 · 程式判讀〕L23102 線性代數（矩陣維度）

閱讀下列程式碼，`print` 的輸出最接近下列何者？

```python
import numpy as np

W = np.random.randn(4, 6)    # 權重矩陣
x = np.random.randn(10, 4)   # 一個 batch 的輸入
h = x @ W
print(h.shape)
```

- [ ] (A) (10, 6)
- [ ] (B) (4, 6)
- [ ] (C) (10, 4)
- [ ] (D) (6, 10)

---

## 第 4 題〔單選〕L23103 數值優化

訓練一個神經網路時，觀察到每個 epoch 的 Loss 不降反升、劇烈震盪甚至發散。最可能的原因與首要調整為何？

- [ ] (A) 學習率（learning rate）設得過高，應優先調降學習率
- [ ] (B) 學習率設得過低，應優先把學習率再調高
- [ ] (C) 訓練資料太多造成干擾，應隨機刪除一半資料
- [ ] (D) 這是正常現象，繼續訓練足夠久必然收斂，不需調整

---

## 第 5 題〔多選（可勾選 4 項）〕L23401 資料與模型治理

為建立可信賴、可追溯的資料與模型治理制度，下列哪些屬於適當作法？（請勾選 4 項）

- [ ] (A) 建立以角色為基礎的資料存取權限控管（RBAC），並保留存取稽核日誌
- [ ] (B) 為模型建立 Model Card，記錄其用途、已知限制、評估結果與適用範圍
- [ ] (C) 僅於導入前做一次性評估，模型上線後不再監控其表現與資料分布
- [ ] (D) 記錄訓練資料的版本與來源（data lineage），確保結果可追溯與可重現
- [ ] (E) 訂定資料保留與刪除政策，對逾期或已無使用目的之個資予以清除
- [ ] (F) 將完整原始個資永久保存並開放全公司自由查閱，以利各單位隨時分析

---

## 第 6 題〔多選（可勾選 3 項）〕L23402 AI 風險與倫理

關於 AI 公平性評估與負責任使用，下列敘述哪些正確？（請勾選 3 項）

- [ ] (A) 群體公平性應以分群指標（如各族群的核准率、真陽性率）衡量，而非只看整體 Accuracy
- [ ] (B) 高風險的自動化決策應提供可解釋性、申訴管道與人為監督（human-in-the-loop）
- [ ] (C) 資料中的歷史偏見（historical bias）可能被模型學習甚至放大，需在資料階段就檢視
- [ ] (D) 只要持續提升模型整體準確率，各族群之間的公平性必然同步改善
- [ ] (E) 發現某族群表現明顯較差時，最快的解法是直接刪除該族群的測試資料

---

## 第 7 題〔多選（可勾選 3 項）〕L23301 數據準備與特徵工程（Data Leakage）

下列哪些做法能正確避免資料洩漏（Data Leakage）？（請勾選 3 項）

- [ ] (A) 先切分 Train/Test，再「僅用訓練集」fit 標準化器，最後才 transform 測試集
- [ ] (B) 在 Train/Test 切分之前，先用整份資料（含測試集）計算平均值與標準差
- [ ] (C) 交叉驗證時，把特徵縮放與編碼放進 Pipeline，於每一折的訓練子集內各自 fit
- [ ] (D) 時間序列資料採「時間順序」切分，避免以未來資料預測過去
- [ ] (E) 加入含有目標答案資訊的特徵（如以「是否已理賠」預測「是否詐欺」）以加速學習

---

## 第 8 題〔情境分析 · 單選〕L23402 AI 風險與倫理（教育與人力資源）

某大學人資部門導入 AI 履歷初篩系統以加速招募。上線後發現女性應徵者被篩掉的比例明顯偏高；進一步調查發現，訓練資料來自過去十年的錄取紀錄，而該期間錄取者以男性為主。下列處理方式何者最恰當？

- [ ] (A) 維持系統現狀，因為它只是忠實複製過去的錄取標準
- [ ] (B) 檢視訓練資料的歷史偏見、以公平性指標分群評估，必要時重新蒐集／平衡資料並加入人為複核與申訴機制
- [ ] (C) 直接把「性別」欄位的值全部改成隨機亂數，即可解決差異
- [ ] (D) 提高整體 Accuracy 門檻，偏見自然會隨之消失

---

## 第 9 題〔情境分析 · 單選〕L23302／L23304 模型選擇與優化（能源與環境監測）

某能源業者建立 PM2.5 空氣品質預測模型，於離線評估時 **訓練 R² = 0.96、驗證 R² = 0.61**，兩者差距很大；模型上線後在新資料上的表現同樣不佳。最可能的問題與首要對策為何？

- [ ] (A) 欠擬合（Underfitting）；應增加模型複雜度與特徵
- [ ] (B) 過擬合（Overfitting）；應加入正則化／Dropout、增加訓練資料或降低模型複雜度，並可採用 Early Stopping
- [ ] (C) 資料漂移（Data Drift）；只要用近期資料重新訓練即可，模型架構不需調整
- [ ] (D) 資料與模型皆無問題，此差距屬正常隨機誤差，可忽略

---

## 第 10 題〔指標計算 · 單選〕L23101 機率／統計（貝氏定理・基率）

某環境監測站的異常排放事件**盛行率為 5%**；感測器在「真的異常」時發出警報的機率為 **80%（敏感度）**，在「正常」時的誤報率為 **10%**。今警報響起，該事件實際為異常的機率最接近下列何者？

- [ ] (A) 約 80%
- [ ] (B) 約 30%
- [ ] (C) 約 5%
- [ ] (D) 約 50%

---

### 作答與批改方式

1. 逐題點擊選項前的核取方塊完成勾選（多選題請勾滿指定項數）。
2. 全部作答完成後，將本 issue 的 label 改為 `ai-analysis`，即會啟動自動批改與錯題分析。
